pkg/handlers/generic/mutation/imageregistries/credentials/inject.go

```diff
@@ -89,31 +83,33 @@ func (h *imageRegistriesPatchHandler) Mutate(
 	}
 
 	// TODO: Add support for multiple registries.
-	if len(imageRegistryCredentials) > 1 {
-		return fmt.Errorf("multiple Image Registry Credentials are not supported at this time")
+	if len(imageRegistries) > 1 {
+		return fmt.Errorf("multiple Image Registry are not supported at this time")
 	}
 
```

Comment:
Do we include a little more feedback for the user, or is this information too low-level? 
```suggestion
	if len(imageRegistries) > 1 {
		return fmt.Errorf("multiple Image Registry are not supported at this time")
	}

```

Reply:
In general, the error messages refer to the "Image Registry variable." Would it be more clear for them to refer to the actual field name, i.e. `imageRegistris`?

Reply:
Agree. it would be good to provide reference to the variable.

Reply:
Fixed. https://github.com/d2iq-labs/capi-runtime-extensions/pull/292/files#diff-384e1aa9d958e44e76b7dff0bc179f92545e0901f68fa402c0eaba7cd2d0cd98R86-R88

Reply:
We can replace this check with openapi validation of the length of the array by specifying `MaxItems: 1` on the image registries definition.